hangman.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
```

Comment:
I like the simple logics and minimum variables approach for this app. Design is clean and simple with notebook background and stick figure drawings. One caveat is lack of restart button which can be implemented by running location.reload() method which will reload the page upon triggering a click event. Keep it up and looking forward to see new projects in the future!